src/common/components/core/TextInput.spec.tsx

```diff
@@ -55,7 +55,7 @@ describe('TextIpnut component', () => {
 
   test('TextInput disabled="false" input', async () => {
     // act
-    render(<TextInput onChange={onChange} value={value} label={label} inset={true} small={true} disabled={false} />);
+    render(<TextInput onChange={onChange} value={value} label={label} inset small disabled={false} />);
```

Comment:
`disabled` is false by default.

Reply:
This will actually require further changes and feels out of scope. `npm run test:coverage' fails with the suggested change.